constraintFunctions.f90

```diff
@@ -149,6 +149,9 @@ subroutine getEnvVarsAtT( t )
     character(len=maxEnvVarNameLength) :: this_env_var_name, orderedEnvVarNames(size( envVarNames ))
     logical :: pressure_set, rh_set, temp_set
 
+    ! TODO, is this the best place to call calcTheta?
```

Comment:
Actually, I'd just move it to below the large comment below - it risks getting lost up here, as it's the only non-declaration statement before the comment.